exercises/E-conditionals/exercise.js

```diff
@@ -9,6 +9,10 @@
 let name = "Daniel";
 let danielsRole = "mentor";
 
+if (name, danielsRole) {
+  console.log("Hi, I'm " + name + ", " + "I'm a " + danielsRole + ".")
```

Comment:
```suggestion
  console.log(`Hi, I'm ${name}, I'm a ${danielsRole}.`); 
```

Reply:
Well done Inna. I learned somethings new in your code.
I suggest to use this model of concatenation instead of using 5 plus.